src/test/scala/com/lightbend/kafkalagexporter/InfluxDBPusherSinkTest.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+ package com.lightbend.kafkalagexporter
+
+ import org.scalatest._
+ import com.typesafe.config.ConfigFactory
+ import scala.collection.JavaConversions.mapAsJavaMap
+ import com.github.fsanaulla.core.testing.configurations.InfluxUDPConf
+ import com.github.fsanaulla.scalatest.embedinflux.EmbeddedInfluxDB
+ import org.scalatest.concurrent.{Eventually, IntegrationPatience}
+ import org.scalatest.{Matchers, TryValues}
+ import sys.process._
+
+class InfluxDBPusherSinkTest extends fixture.FreeSpec with Matchers
+    with EmbeddedInfluxDB
+    with InfluxUDPConf
+    with TryValues
+    with Eventually
+    with IntegrationPatience {
+
+  case class Fixture(properties: Map[String, Any], port: Int)
+  type FixtureParam = Fixture
+
+  override def withFixture(test: OneArgTest): Outcome = {
+    val port: Int = 8086
+    val properties: Map[String, Any] = Map(
+      "reporters.influxDB.endpoint" -> "http://localhost",
+      "reporters.influxDB.port" -> port
+    )
+    test(Fixture(properties, port))
+  }
+
+  "InfluxDBPusherSinkImpl should" - {
+
+    "create database" in { fixture =>
+      val sink = InfluxDBPusherSink(new InfluxDBPusherSinkConfig("InfluxDBPusherSink", List("kafka_consumergroup_group_max_lag"), ConfigFactory.parseMap(mapAsJavaMap(fixture.properties))), Map("cluster" -> Map.empty))
+      Thread.sleep(1000)
+      val port = fixture.port
+      val url = s"http://localhost:$port"
+      val query = "SHOW DATABASES"
+      val cmd = Seq("curl", "-G", s"$url/query", "--data-urlencode", s"q=$query")
+      val output = cmd.!!
```

Comment:
You could use a ScalaTest `eventually` here (and wait for a zero exit code, or a 200 result?) instead of a sleep.

Reply:
Yes, this is also updated accordingly.